bgpd/bgp_debug.c

```diff
@@ -1557,6 +1559,91 @@ DEFUN (no_debug_bgp_update_groups,
 	return CMD_SUCCESS;
 }
 
+DEFUN (debug_bgp_vpn,
+       debug_bgp_vpn_cmd,
+       "debug bgp vpn <leak-from-vrf|leak-to-vrf|rmap-event|label>",
+       DEBUG_STR
+       BGP_STR
+       "VPN routes\n"
+       "leaked from vrf to vpn\n"
+       "leaked to vrf from vpn\n"
+       "route-map updates\n"
+       "labels\n")
+{
+	if (!strcmp(argv[3]->arg, "leak-from-vrf")) {
```

Comment:
This breaks autocompletion. If a user enters `debug bgp vpn leak-f` the command will be accepted but will fail. Compare against `->text`. This is documented in `doc/cli.md`.

Correct:
`+	if (!strcmp(argv[3]->text, "leak-from-vrf")) {`

Better:
```
int idx = 0;
...
+	if (argv_find(argv, argc, "leak-from-vrf", &idx) {
```

Reply:
Thank you for catching these. The CLI internals have changed since last time I worked on frr, so I'm playing catch-up.